packages/common-types/src/node-protocol.ts

```diff
@@ -43,7 +43,7 @@ export namespace Node {
     UNINSTALL = "uninstallEvent",
     PROPOSE_STATE = "proposeStateEvent",
     REJECT_STATE = "rejectStateEvent",
-    MULTISIG_CREATED = "multisigCreatedEvent"
+    CREATE_MULTISIG = "createMultisigEvent"
```

Comment:
does this enum _need_ to use strings ?

Reply:
I mean preferably yes. Otherwise the node would have to send integer enum values over the wire, which is much less debuggable